src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -34,6 +34,11 @@
     <Compile Remove="commands\utility\ShowCommand\ShowCommandProxy.cs" />
     <Compile Remove="commands\utility\update-list.cs" />
     <Compile Remove="singleshell\installer\MshUtilityMshSnapin.cs" />
+    <Compile Remove="commands\utility\WebCmdlet\Common\BasicHtmlWebResponseObject.Common.cs" />
```

Comment:
Why are we removing these files?

Reply:
If the files don't exist we should remove these enties.

Reply:
I have removed them.